### ISSUE TITLE
Remove the "doNotClean" message from all masterlists

### ISSUE THREAD

**Issue by pStyl3** [loot/loot.github.io#89]
Plugins with ITMs (Identical to Master), Deleted References or Deleted Navmeshes are considered `dirty` and we advise users to clean those plugins with xEdit.

However, there are some plugins where mod authors are of the opinion, that those ITMs are `intentional` and as such, shouldn't be cleaned.

If we find such a plugin, that only has ITMs (which are claimed to be intentional) and no other problems (like Deleted References and Deleted Navmeshes), we have a couple of ways to handle the situation:

- Don't actively tell our users about the cleaning report
- Report the identified ITMs nonetheless
- Tell our users to not clean the plugin in question (`doNotClean`)

As of today, the masterlists have the following number of plugins with the `doNotClean` message attached:
Skyrim: 48
SSE: 4
Oblivion: 31
Morrowind: 0
FO3: 1
FO4: 0
FNV: 8

92 uses of `doNotClean`.

From a strictly technical point of view, the option to report ITMs nonetheless is the sentiment that I would likely share the most. It has been stated by multiple people, that ITMs aren't needed to achieve some change, so in other words, that "Intentional ITMs" do not really have valid use cases, or are the best way of approaching things.

From the point of view of LOOT, it isn't quite that easy though. As of today, we provide the ability to tell users to `doNotClean` a mod. In my understanding, this has been made practise so that mod authors do have a say in the matter. While we trust xEdit, there are people who may disagree, so this possibility opens up the situation for them.

That being said, I'm leaning more and more towards removing the `doNotClean` messages altogether. I'd rather give mod authors of plugins which only have ITMs and no other problems, where they claim those ITMs are intentional, the option that we don't actively tell users about those ITMs, than telling the users to not clean the plugins. That way we can keep a middle ground of some sorts, but don't further act on the narrative that cleaning "intentional" ITMs can lead to issues.

So in conclusion, I think we should remove the `doNotClean` messages. Plugins that only have ITMs and where those ITMs are claimed to be intentional could be left "untouched" by the masterlist, if the mod author so wishes, but don't actively tell users to not clean plugins. Thoughs?
